packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -368,10 +369,11 @@ export class ConnectionFlow {
       this.trigger = trigger
       this.account = account
       this.konnector = konnector
+
       this.t = t
 
       assert(client, 'No client')
-      const konnectorPolicy = findKonnectorPolicy(konnector)
+      const konnectorPolicy = findKonnectorPolicy(this.konnector)
```

Comment:
Why do we need this change? It seems that `handleFormSubmit` is call with options.konnector (https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/components/TriggerManager.jsx#L140) 

Sometimes ConnectionFlow is instanciated without `konnector` in the constructor https://github.com/cozy/cozy-libs/blob/ba70b8fb5b7b95196d3bec51942c3a56410712c1/packages/cozy-harvest-lib/src/cli/cli.js#L69 so using this.konnector will result in breaking at least this method

Reply:
I remember there are some cases where options.konnector is null. I am looking for it at the moment.

Reply:
I could not reproduce the problem and I can't see why option.konnector would be null by reading the code. I remove this change.